src/base/http/connection.cpp

```diff
@@ -1,177 +0,0 @@
-/*
- * Bittorrent Client using Qt and libtorrent.
- * Copyright (C) 2018  Mike Tzou (Chocobo1)
- * Copyright (C) 2014  Vladimir Golovnev <glassez@yandex.ru>
- * Copyright (C) 2006  Ishan Arora and Christophe Dumez <chris@qbittorrent.org>
```

Comment:
And suddenly these copyright holders just give up their rights and you can claim all the code, do I read you correctly?